prettier/src/printHubl.ts

```diff
@@ -52,6 +52,18 @@ const printTagArgs = (node) => {
   });
 };
 
+const printJsonBody = (node) => {
+  try {
+    // This is a predictable tag structure
+    const bodyText = node.children[0].children[0].value;
+    const formattedBodyText = format(bodyText, { parser: "json" });
+    return [line, join(line, formattedBodyText.split("\n"))];
```

Comment:
One important thing we want to do here is replace any newlines with a prettier `line` (or `softline`, `hardline` from https://github.com/prettier/prettier/blob/main/commands.md)